lib/core-integration/src/Test/Integration/Scenario/API/Network.hs

```diff
@@ -51,14 +53,32 @@ import qualified Network.HTTP.Types.Status as HTTP
 spec :: forall t. SpecWith (Context t)
 spec = do
     it "NETWORK - Can query network information" $ \ctx -> do
-        eventually_ $ do
+        r <- eventually $ do
             now <- liftIO getCurrentTime
             r <- request @ApiNetworkInformation ctx networkInfoEp Default Empty
             expectResponseCode @IO HTTP.status200 r
             verify r
                 [ expectFieldSatisfy nextEpoch ((> now) . epochStartTime)
```

Comment:
I am not sure if there can be more strict to way to check this... (To make sure that `epochStartTime` is really the the exact time new epoch starts)